.github/workflows/release_nightly.yml

```diff
@@ -416,12 +417,12 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - name: Package selfhosted
-        if: ${{ !matrix.demo }}
+        if: matrix.demo
         run: zip -r "${{ needs.create-nightly-release.outputs.package_prefix }}-web-selfhosted.zip" .
         working-directory: web/packages/selfhosted/dist
 
       - name: Upload selfhosted
-        if: ${{ !matrix.demo }}
+        if: matrix.demo
```

Comment:
Why is the negation removed here? 🤔

Reply:
So selhosted continues working on Safari 16.3-. It's also why the workflows long name is changed to "Build web demo and selfhosted"

Reply:
See the second paragraph of https://github.com/ruffle-rs/ruffle/pull/19316#issue-2809964632

Reply:
Then, since it now means "demo and selfhosted", maybe this flag should be replaced with a different flag with inverse values, for "extensions"? :thinking:

Reply:
Oh but it should be clear that this flag governs "whether we're building the browser extensions, or everything else", and not "whether WASM extensions are enabled".

Reply:
Hmm, looking at it again, more closely: Since (almost) _all_ steps of the `build-web` job in the `Release Nightly` workflow are gated by the value of `matrix.demo`, maybe it would make more sense to just split it, getting rid of all the conditionals? :thinking:

Reply:
Sure, a couple of setup steps at the beginning will have to be duplicated this way, but maybe the increased clarity is worth it...?
BTW still waiting for https://github.com/actions/runner/issues/1182 or something similar...

Reply:
Something like this: [`3299304` (#19767)](https://github.com/ruffle-rs/ruffle/pull/19767/commits/3299304031f9db97dab46a5594b2aa99039add35)